LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -3818,6 +3818,54 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
 
                         break;
 
+                    case Model.ROG_STRIX_Z790_E_GAMING_WIFI: //NCT6798D
+
+                        // Volts
+                        v.Add(new Voltage("CPU Core Voltage", 0));
+                        v.Add(new Voltage("+5V", 1));
+                        v.Add(new Voltage("AV (Standby)", 2));
+                        v.Add(new Voltage("+3.3V", 3));
+                        v.Add(new Voltage("+12V", 4));
+                        v.Add(new Voltage("AUXTIN4 (VIN8)", 5));
```

Comment:
Based off the ASUS boards I've tested including the Z790-A, this tends to be `IVR Atom L2 Cluster #1`. Worth mentioning since this entire data entry is super similar to the Z790-A and Dark Hero layout.

Reply:
Will do.